components/builder-originsrv/src/migrations/2018-08-24-184520_initial_originsrv_migration/up.sql

```diff
@@ -0,0 +1,1149 @@
+    integration_id bigint NOT NULL REFERENCES origin_integrations(id) ON DELETE CASCADE,
+    UNIQUE (project_id, integration_id)
+);
+
+CREATE TABLE origin_public_encryption_keys (
+    id bigint DEFAULT next_id_v1('origin_public_key_id_seq') PRIMARY KEY NOT NULL,
+    origin_id bigint REFERENCES origins(id),
+    owner_id bigint,
+    name text,
+    revision text,
+    full_name text UNIQUE,
+    body bytea,
+    created_at timestamp with time zone DEFAULT now(),
+    updated_at timestamp with time zone DEFAULT now()
+);
+
+CREATE TABLE origin_public_keys (
+    id bigint DEFAULT next_id_v1('origin_public_key_id_seq') PRIMARY KEY NOT NULL,
+    origin_id bigint REFERENCES origins(id),
+    owner_id bigint,
+    name text,
+    revision text,
+    full_name text UNIQUE,
+    body bytea,
+    created_at timestamp with time zone DEFAULT now(),
+    updated_at timestamp with time zone DEFAULT now()
+);
+
+CREATE TABLE origin_secret_keys (
+    id bigint DEFAULT next_id_v1('origin_secret_key_id_seq') PRIMARY KEY NOT NULL,
+    origin_id bigint REFERENCES origins(id),
+    owner_id bigint,
+    name text,
+    revision text,
+    full_name text UNIQUE,
+    body bytea,
+    created_at timestamp with time zone DEFAULT now(),
+    updated_at timestamp with time zone DEFAULT now()
+);
+
+CREATE TABLE origin_channel_packages (
+    channel_id bigint NOT NULL REFERENCES origin_channels(id) ON DELETE CASCADE,
+    package_id bigint NOT NULL REFERENCES origin_packages(id),
+    created_at timestamp with time zone DEFAULT now(),
+    updated_at timestamp with time zone DEFAULT now(),
+    PRIMARY KEY (channel_id, package_id)
+);
+
+CREATE TABLE origin_members (
+    origin_id bigint NOT NULL REFERENCES origins(id),
+    origin_name text,
+    account_id bigint NOT NULL,
+    account_name text,
+    created_at timestamp with time zone DEFAULT now(),
+    updated_at timestamp with time zone DEFAULT now(),
+    PRIMARY KEY (origin_id, account_id)
+);
+
+CREATE FUNCTION accept_origin_invitation_v1(oi_invite_id bigint, oi_ignore boolean) RETURNS void
+    LANGUAGE plpgsql
+    AS $$
+  DECLARE
+    oi_origin_id bigint;
+    oi_origin_name text;
+    oi_account_id bigint;
+    oi_account_name text;
+  BEGIN
+    IF oi_ignore = true THEN
+      UPDATE origin_invitations SET ignored = true, updated_at = now() WHERE id = oi_invite_id;
+    ELSE
+      SELECT origin_id, origin_name, account_id, account_name INTO oi_origin_id, oi_origin_name, oi_account_id, oi_account_name FROM origin_invitations WHERE id = oi_invite_id;
+      PERFORM insert_origin_member_v1(oi_origin_id, oi_origin_name, oi_account_id, oi_account_name);
+      DELETE FROM origin_invitations WHERE id = oi_invite_id;
+    END IF;
+  END
+$$;
+
+CREATE FUNCTION add_audit_entry_v1(p_origin_id bigint, p_package_id bigint, p_channel_id bigint, p_operation smallint, p_trigger smallint, p_requester_id bigint, p_requester_name text) RETURNS SETOF audit
```

Comment:
we can also remove this..

note: jobsrv has audit table/functions with same name (audit/add_audit_entry_v1), so removing these here will make it easier to migrate jobsrv over if we want to keep the same schema